CONTRIBUTING.md

```diff
@@ -1,5 +1,18 @@
 # Contributing to the Cardano Ledger
 
+## Roles and responsibilities
+
+The
+[@cardano-ledger](https://github.com/orgs/input-output-hk/teams/cardano-ledger)
+group is responsible for
+helping with reviewing and merging pull requests, adjudicating technical (or other) disputes,
+releasing the ledger packages on [CHaP](https://github.com/input-output-hk/cardano-haskell-packages).
+
+@hamishmack can help with issues regarding this repository's continuous integration and nix infrastructure.
```

Comment:
@hamishmack - I'd like to double check with you that this is okay.

Reply:
Yes, I will be happy to help.

Reply:
thank you!!